tests/integration/conftest.py

```diff
@@ -38,18 +38,53 @@ def pytest_addoption(parser):
     parser.addoption(
         "--runslow", action="store_true", default=False, help="run slow tests"
     )
+    parser.addoption(
+        "--runtier1",
```

Comment:
This is not required.

`pytest -m tier1` could help.

You just add tier2 marker to test case which does not tier1 marker.

Reply:
Something like:

```python
def pytest_collection_modifyitems((config, items):
    for item in items:
        # Set test as tier 2 if not tier assigned.
        if (
            "tier1" not in item.keywords
        ):
            item.add_marker(pytest.mark.tier2)

```

Reply:
Done, thanks!